ansible_collections/arista/cvp/docs/how-to/cvp-authentication.md

```diff
@@ -45,7 +45,8 @@ ansible_network_os: eos
 ```
 
 !!! note
-    Vault encrypted variables are not supported as password yet. Use ansible vault file instead. https://docs.ansible.com/ansible/latest/user_guide/vault.html#encrypting-files-with-ansible-vault
+    Depending on the ansible version, vault encrypted variables may not supported because of https://github.com/ansible/ansible/issues/75503. ansible-cvp code will check if the provided password (and password only) is malformed and inform the user by raising an exception.
```

Comment:
just a nit `may not supported` --> `may not be supported`

Reply:
fixed in cb34ebb

Thanks!